finality-aleph/src/sync/mod.rs

```diff
@@ -6,6 +6,8 @@ use std::{
 mod substrate;
 mod ticker;
 
+const LOG_TARGET: &str = "aleph-block-sync";
```

Comment:
Heh, strictly speaking this is unused, but I guess we have to add it in _a_ PR, so this one is as good as any. :P

Reply:
Oh no :O this is from other PR :O